docs/_posts/2023-03-29-functional-modularity-1.md

```diff
@@ -197,12 +197,12 @@ Thanks for taking this journey with me. Questions? Comments? I would love to con
       
       I think Hughes stayed silent on the topic of OOP modularity because it is an independent idea to his thesis about the importance of functional programs. Object-oriented and functional paradigms coexist just fine, and the modularity they provide is complimentary. In fact, Scala 3 which we are using for this post's hands on exercises is a great example of one that emphasizes object-oriented and functional paradigms working together. 
       
-      If Hughes was warning readers off of any particular paradigm it would likely be the imperative paradigm, which many advocates of the functional style believe is only appropriate in circumstances where the functional approach can't be used, or when it complicates code or hurts its runtime performance. For a thorough treatment of functional as it compares to imperative coding in an ideological sense, one of the great examinations of this topic can be found in [John Backus's Turing Award Lecture](https://dl.acm.org/doi/10.1145/359576.359579). Martin Odersky, the original creator of Scala, has historically taken a more even-handed viewpoint, and in the [keynote to the latest scalacon 2022](https://www.youtube.com/watch?v=QRcD9Zc7eq4&t=2164s&ab_channel=ScalaCon), he expressed the importance of not treating imperative and functional styles as "two absolutes" but instead as a "spectrum", where depending on the context some amount of both paradigms is warranted.
+      If Hughes was warning readers off of any particular paradigm it would likely be the imperative paradigm, which many advocates of the functional style believe is only appropriate in circumstances where the functional approach can't be used, or when it complicates code or hurts its runtime performance. For a thorough treatment of functional as it compares to imperative coding in an ideological sense, one of the great examinations of this topic can be found in [John Backus's Turing Award Lecture](<https://dl.acm.org/doi/10.1145/359576.359579>). Martin Odersky, the original creator of Scala, has historically taken a more even-handed viewpoint, and in the [keynote to the latest scalacon 2022](<https://www.youtube.com/watch?v=QRcD9Zc7eq4&t=2164s&ab_channel=ScalaCon>), he expressed the importance of not treating imperative and functional styles as "two absolutes" but instead as a "spectrum", where depending on the context some amount of both paradigms is warranted.
```

Comment:
```suggestion
      If Hughes was warning readers off of any particular paradigm it would likely be the imperative paradigm, which many advocates of the functional style believe is only appropriate in circumstances where the functional approach can't be used, or when it complicates code or hurts its runtime performance. For a thorough treatment of functional as it compares to imperative coding in an ideological sense, one of the great examinations of this topic can be found in [John Backus's Turing Award Lecture](https://dl.acm.org/doi/10.1145/359576.359579). Martin Odersky, the original creator of Scala, has historically taken a more even-handed viewpoint, and in the [keynote to the latest scalacon 2022](https://www.youtube.com/watch?v=QRcD9Zc7eq4&t=2164s&ab_channel=ScalaCon), he expressed the importance of not treating imperative and functional styles as "two absolutes" but instead as a "spectrum", where depending on the context some amount of both paradigms is warranted.
```

ℹ️ FYI, the usage of bracketed URLs in markdown is typically for when you need to represent a URL hyperlink that doesn't have its own text. E.g. to avoid saying `[SeanKilleen.com](https://SeanKilleen.com)`, I could use `<https://SeanKilleen.com>` instead.